submissions/d-pozniak/html-css-popup/index.html

```diff
@@ -0,0 +1,420 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>POPUP</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+</head>
+<body>
+<div class="header">
```

Comment:
div for page header? 
![image](https://user-images.githubusercontent.com/16196199/190488859-81b506ea-1f02-4ccf-8945-73221db8c43c.png)

[header](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)